app/lib/tenejo/preflight.rb

```diff
@@ -34,7 +35,7 @@ class PFFile < PreFlightObj
     end
     validates_each :resource_type do |rec, att, val|
       if val.present?
-        rec.errors.add(att, "Resource type #{val} is not recognized and will be left blank.") unless RESOURCE_TYPES["terms"].map { |x| x["term"] }.include?(val)
+        rec.errors.add(att, "Resource type \"#{val}\" on line #{rec.lineno} is not recognized and will be left blank.") unless RESOURCE_TYPES["terms"].map { |x| x["term"] }.include?(val)
```

Comment:
A bad resource type should be a warning instead of an error.  So I'm thinking this could should look more like the license and rights code - i.e. shouldn't this be `rec.warnings.add`